### PR TITLE
fix: update error highlight border in RTE widget

### LINKED ISSUE #14019: [Bug]-[62]:RTE error highlight includes the label element
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The RTE widget error highlight is on both the component and label. Whereas it should only be on the input component.

While fixing this we need to find the same for any I/O widgets with label. 

Screenshot 2022-05-20 at 15 34 59

### Steps To Reproduce

1. D&D RTE
2. Set to required
3. check the error highlight; it must be only on the input component

### Public Sample App

_No response_

### Version

Cloud - v1.7.0

### PR DESCRIPTION
**Pull request by techbhavin**
## Description

> In RTE Widget Remove Error Highlight from the label and apply only on RTE input.

Fixes #14019

## Type of change

- Bugfix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/issue-14019 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 56.71 **(0)** | 38.66 **(-0.01)** | 36.23 **(0)** | 56.96 **(0)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**</details>

### CODE REVIEW

File: app/client/src/widgets/RichTextEditorWidget/component/index.tsx [hunk 1]
```diff
@@ -44,11 +41,14 @@ const StyledRTEditor = styled.div<{
...
 
-export const RichTextEditorInputWrapper = styled.div`
+export const RichTextEditorInputWrapper = styled.div<{
+  isValid?: boolean;
+}>`
   display: flex;
   width: 100%;
   min-width: 0;
   height: 100%;
+  border: 1px solid ${(props) => (props.isValid ? "none" : Colors.DANGER_SOLID)};
```
Comment (Tooluloope) [inline app/client/src/widgets/RichTextEditorWidget/component/index.tsx:51]: Could you add a cypress test for this, so it doesn't reoccur

**Review by Tooluloope — CHANGES_REQUESTED** [review]

**Review by Tooluloope — DISMISSED** [review]

**Comment by shadabbuchh** [comment]
This issue if fixed. The error (red) highlight now appears only on the input component of the RTE widget.


However @Tooluloope / @techbhavin I notice that the error highlight does not adjust with the changing `Border Radius`. Is that something that is expected?

**Comment by shadabbuchh** [comment]
The issue is fixed. Moving it to `Done`.

**Pull request merged**